File: ping-js-bot/src/bot.js
require("dotenv").config();

console.log(process.env.PING_JS_BOT_TOKEN);

const {Client} = require("discord.js");
const client = new Client();

client.on("ready", () => {
    console.log(`${client.user.username} is connected`);
});

client.on("message", (message) => {
    if(message.author.bot) {
        return ;
    } else {
        if(message.content === "hello") {
            message.channel.send("hello");
        }
    }
})

client.login(process.env.PING_JS_BOT_TOKEN).then(r => console.log(r));

File: ping-py-bot/src/bot.py
import os
from os.path import join, dirname

from discord import Client
from dotenv import load_dotenv
import discord

client: Client = discord.Client()
dotenv_path: str = join(dirname(__file__), '../.env')
load_dotenv(dotenv_path)

PING_PY_BOT_TOKEN: str = os.environ.get('PING_PY_BOT_TOKEN')


@client.event
async def on_ready():
    print('We have logged in as {0.user}'.format(client))


@client.event
async def on_message(message):
    print("message received")
    if message.author == client.user:
        return

    if message.content.lower().find('hello') != -1:
        await message.channel.send('HELLO!')

    if message.content.lower().find('ping') != -1:
        await message.channel.send('PONG!')

print(PING_PY_BOT_TOKEN)
client.run(PING_PY_BOT_TOKEN)
